web/packages/teleport/src/Integrations/EditAwsOidcIntegrationDialog.tsx

```diff
@@ -146,9 +146,8 @@ export function EditAwsOidcIntegrationDialog(props: Props) {
                     s3Prefix={s3Prefix}
                   />
                   <OutlineWarn>
-                    Next reconfiguration will remove this integrations S3 bucket
-                    automatically. Storing the OpenID configuration in a S3
-                    bucket is deprecated.
+                    Reconfiguring is suggested to remove the deprecated S3
+                    fields from this integration.
```

Comment:
I'm not 100% sure I understand what this is saying. I'm sure there is even better copy, but I'll try a suggestion and you can decide what you like
"We suggest reconfiguring this integration to remove the deprecated s3 fields"